climakitae/core/constants.py

```diff
@@ -0,0 +1,18 @@
+# constants.py
+"""This module defines package level constants"""
```

Comment:
Big fan of this reorganizing! The constants module is a great idea

Reply:
> Big fan of this reorganizing! The constants module is a great idea

Thanks! Yeah, `pkg_resource` is deprecated and going away...